bin/rialto/runtime/src/lib.rs

```diff
@@ -463,7 +464,7 @@ construct_runtime!(
 		BridgeKovanCurrencyExchange: pallet_bridge_currency_exchange::<Instance2>::{Module, Call},
 		BridgeMillau: pallet_substrate_bridge::{Module, Call, Storage, Config<T>},
 		BridgeCallDispatch: pallet_bridge_call_dispatch::{Module, Event<T>},
-		BridgeMillauMessageLane: pallet_message_lane::{Module, Call, Event<T>},
+		BridgeRialtoMessageLane: pallet_message_lane::{Module, Call, Event<T>},
```

Comment:
Why do we have `BridgeRialto` in the `rialto` runtime? I thought we would rather have `Millau` here, no? (we do have `BridgeMillau` for instance)

Reply:
I can't say why I've changed that :/